geeksurvey/templates/base.html

```diff
@@ -5,9 +5,18 @@
 <title>GeekSurvey</title>
 
 <link rel="shortcut icon" href="{% static 'gkslogo.png' %}" type="image/x-icon">
+<!--
 <link rel="stylesheet" type="text/css" href="{% static 'style.css' %}">
+-->
+
+<meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Shouldn't need the jquery script as that was used for Bootstrap versions prior to v5 (v5 decouples from jquery). The JS script bundle on line 19 should be all that is needed as it includes the vanilla JS and Popper stuff needed by Bootstrap v5. Could also just comment this line out temporarily.